index.html

```diff
@@ -109,6 +109,18 @@ <h2 class="f3">Courses</h2>
     </div>
   </div>
 
+  <div class="mv3">
+    <h2 class="f3 mt0">Participate</h2>
+    <p>
```

Comment:
```suggestion
    <p class="measure-wide">
```

improves readability

Reply:
How's this?

<img width="1038" alt="Screen Shot 2021-03-05 at 11 23 28 AM" src="https://user-images.githubusercontent.com/13065028/110143396-378d1200-7da5-11eb-8bda-5f6d3d1d1b10.png">

Reply:
It's more readable although the balance is a bit off now but I guess that's a trade-off we have to make.